browser/ui/views/frame/brave_browser_view.cc

```diff
@@ -262,10 +303,9 @@ void BraveBrowserView::UpdateSideBarHorizontalAlignment() {
       prefs::kSidePanelHorizontalAlignment);
 
   sidebar_container_view_->SetSidebarOnLeft(on_left);
-  static_cast<BraveContentsLayoutManager*>(GetContentsLayoutManager())
-      ->set_sidebar_on_left(on_left);
+  GetBrowserViewLayout()->set_sidebar_on_left(on_left);
```

Comment:
nit: How about call `sidebar_container_view_->SetSidebarOnLeft(on_left);` from `BraveBrowserViewLayout::SetSidebarOnLeft()` as layout manager manages sidebar container?

Reply:
Good idea.

Another option would be to have `BraveBrowserViewLayout` just get the "on left" state from `sidebar_container_view_`, like `sidebar_container_view_->is_sidebar_on_left()`. That way, we wouldn't have to worry about the "on left" state ever getting out of sync. Would that work?

Reply:
I think it's good idea. It seems `BraveBrowserView` itself doesn't need left/right info

Reply:
Seems like `BraveBrowserView` still needs to listen to the pref change so that it can re-layout, though?

Reply:
Ah, I thoughtyou'll use container view for listening prefs instead :) 
IMO, current one seems fine but container view or layout manager also could listen can listen prefs change if you want.